adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1422,7 +1422,7 @@ public static String getVersionName() {
         // Package manager does not report for ADAL
         // AndroidManifest files are not merged, so it is returning hard coded
         // value
-        return "1.15.1";
+        return "1.15.1-hf1";
```

Comment:
Fine for AAD / ESTS right? and their versioning? Since we shipped with -RC I assume this is fine as it would have blocked TrueMAM CA.

Reply:
Not aware of any issues -- @addev-ashish we're cool, yeah?